### ISSUE TITLE
Mechanism Quality for Droid Storage Compartment -999999

### ISSUE THREAD

**Issue by cyberops2021** [swgemu/Core3#11]
I seem to have found a bug while crafting a Droid Storage Compartment. If you can point me to the function responsible for computing the Mechanism Quality, I would be happy to look into it.
![image](https://github.com/swgemu/Core3/assets/96167343/58437b3a-1e2a-4dbe-8fb8-57ac42591847)

**Comment by Hakry** [comment]
All of our bug reprots are handled via Mantis Bug Tracker. This issue can be found here [https://www.swgemu.com/bugs/view.php?id=8868](https://www.swgemu.com/bugs/view.php?id=8868)


Attribute values default to that when one is not set from the item template, the AttributeMap puts that in as a default. For sure can look into it, would be appreciate. Push the fix to [Gerrit](https://review.swgemu.com) for review

**Comment by cyberops2021** [comment]
Would you mind telling me what the minimum and maximum values when experimenting are supposed to be?

**Comment by cyberops2021** [comment]
I haven't fixed it yet, but I'm on to something. This schematic appears to be the only droid schematic with "usemodifier" as an experimentalSubGroup item. I also noticed this lua is missing "mechanism_quality" as an experimentalSubGroup while others had it.

**Comment by cyberops2021** [comment]
I don't believe that I have sign in privileges ... but here is what I think the fix is ...

I believe this can be solved by editing 
droid_storage_compartment.lua:51:       experimentalSubGroupTitles = {"null", "null", "decayrate", "hitpoints", "usemodifier"},

and changing "usemodifier" to "mechanism_quality"
![image](https://github.com/swgemu/Core3/assets/96167343/7a18f682-89ac-40ca-a098-84af3877f1ae)
